INSTALL.md

```diff
@@ -126,6 +126,14 @@ set -g @dracula-powerline-bg-transparent true
 set -g @dracula-inverse-divider 
 ```
 
+#### [color theming](https://github.com/dracula/tmux/blob/master/docs/color_theming/README.md)
```

Comment:
This needs to be a relative link instead of an absolute link: https://stackoverflow.com/questions/7653483/github-relative-link-in-markdown-file